dto/company.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty } from "class-validator";
+
+
+export class CompanyDTO {
+
+    @IsNotEmpty({ message: 'Company name is required' })
+    name!: string;
+
+    @IsNotEmpty({ message: 'Contact is required' })
+    contact!: string;
+
+    @IsNotEmpty({ message: 'Details are required' })
+    details!: string;

```

Comment:
i don't understand the purpose of this field? There should be meaningful fields in this as this is the most important entity. For example, there should be an embedded entity of address having country, city, province, address, city, zip code and then you can have website, email address, phone number, status(active/inactive), createdAt, updatedAt,